### PR TITLE
Update hexchat snapcraft.yml

### PR DESCRIPTION
**Pull request by diddledan**
Update for smaller set of dependencies and tidy-up in-lined options. Also changed the snap build to using the released version tarball rather than the tip of git.

I've published a snap based on this configuration to the store for anyone to check-out: https://uappexplorer.com/app/unofficial-hexchat.diddledan
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/ubuntu/snappy-playpen/pull/187%23discussion_r71658187%22%2C%20%22https%3A//github.com/ubuntu/snappy-playpen/pull/187%23discussion_r71658240%22%2C%20%22https%3A//github.com/ubuntu/snappy-playpen/pull/187%23discussion_r71658398%22%2C%20%22https%3A//github.com/ubuntu/snappy-playpen/pull/187%23discussion_r71718598%22%2C%20%22https%3A//github.com/ubuntu/snappy-playpen/pull/187%23discussion_r71718603%22%2C%20%22https%3A//github.com/ubuntu/snappy-playpen/pull/187%23discussion_r71718607%22%2C%20%22https%3A//github.com/ubuntu/snappy-playpen/pull/187%23issuecomment-234480715%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ubuntu/snappy-playpen/pull/187%23issuecomment-234480715%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Nice%20work%21%22%2C%20%22created_at%22%3A%20%222016-07-22T08%3A07%3A09Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1346979%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dholbach%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%2009da892a4a4d865749c4ddf9fcf3872fbd80d50c%20hexchat/snapcraft.yaml%204%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ubuntu/snappy-playpen/pull/187%23discussion_r71658187%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22We%20have%20as%20best%20practice%20to%20enforce%20version%20being%20in%20a%20string%20field.%20%28that%20prevents%20if%20we%20update%20to%20%5C%222.20%5C%22%20to%20be%20translated%20in%20%5C%222.2%5C%22.%20Please%20revert%20that%20change.%22%2C%20%22created_at%22%3A%20%222016-07-21T07%3A26%3A00Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1823296%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/didrocks%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222016-07-21T14%3A42%3A55Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/147548%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/diddledan%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20hexchat/snapcraft.yaml%3AL1-7%22%7D%2C%20%22Pull%2009da892a4a4d865749c4ddf9fcf3872fbd80d50c%20hexchat/snapcraft.yaml%2020%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ubuntu/snappy-playpen/pull/187%23discussion_r71658240%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22This%20was%20less%20than%2080%20characters%2C%20same%2C%20as%20per%20our%20best%20practices%2C%20please%20leave%20current%20list%20in%20one%20line%20as%20it%20is%20for%20clarity.%22%2C%20%22created_at%22%3A%20%222016-07-21T07%3A26%3A33Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1823296%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/didrocks%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222016-07-21T14%3A42%3A56Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/147548%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/diddledan%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20hexchat/snapcraft.yaml%3AL11-37%22%7D%2C%20%22Pull%2009da892a4a4d865749c4ddf9fcf3872fbd80d50c%20hexchat/snapcraft.yaml%2022%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ubuntu/snappy-playpen/pull/187%23discussion_r71658398%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Same%2C%20as%20part%20of%20our%20best%20practices%20%28sorry%2C%20that%20should%20be%20documented%29%2C%20the%20order%20here%20should%20be%3A%5Cr%5Cn%60%60%60%5Cr%5Cn%20%20source%3A%5Cr%5Cn%20%20plugin%3A%5Cr%5Cn%20%20snap%3A%5Cr%5Cn%20%20build-packages%3A%5Cr%5Cn%20%20after%3A%5Cr%5Cn%60%60%60%5Cr%5CnWe%20don%27t%20order%20them%20in%20ascii%20order%2C%20but%20in%20a%20logical%20one.%22%2C%20%22created_at%22%3A%20%222016-07-21T07%3A27%3A59Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1823296%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/didrocks%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-complete%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222016-07-21T14%3A42%3A56Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/147548%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/diddledan%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20hexchat/snapcraft.yaml%3AL11-37%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/ubuntu/snappy-playpen/pull/187#issuecomment-234480715'>General Comment</a></b>
- <a href='https://github.com/dholbach'></a> Nice work!
- [x] <a href='#crh-comment-Pull 09da892a4a4d865749c4ddf9fcf3872fbd80d50c hexchat/snapcraft.yaml 4'></a> &nbsp;<b><a href='https://github.com/ubuntu/snappy-playpen/pull/187#discussion_r71658187'>File: hexchat/snapcraft.yaml:L1-7</a></b>
- <a href='https://github.com/didrocks'></a> We have as best practice to enforce version being in a string field. (that prevents if we update to "2.20" to be translated in "2.2". Please revert that change.
- [x] <a href='#crh-comment-Pull 09da892a4a4d865749c4ddf9fcf3872fbd80d50c hexchat/snapcraft.yaml 20'></a> &nbsp;<b><a href='https://github.com/ubuntu/snappy-playpen/pull/187#discussion_r71658240'>File: hexchat/snapcraft.yaml:L11-37</a></b>
- <a href='https://github.com/didrocks'></a> This was less than 80 characters, same, as per our best practices, please leave current list in one line as it is for clarity.
- [x] <a href='#crh-comment-Pull 09da892a4a4d865749c4ddf9fcf3872fbd80d50c hexchat/snapcraft.yaml 22'></a> &nbsp;<b><a href='https://github.com/ubuntu/snappy-playpen/pull/187#discussion_r71658398'>File: hexchat/snapcraft.yaml:L11-37</a></b>
- <a href='https://github.com/didrocks'></a> Same, as part of our best practices (sorry, that should be documented), the order here should be:
```
source:
plugin:
snap:
build-packages:
after:
```
We don't order them in ascii order, but in a logical one.

<a href='https://www.codereviewhub.com/ubuntu/snappy-playpen/pull/187?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/ubuntu/snappy-playpen/pull/187?approve=1'></a>&nbsp;<a href='https://github.com/ubuntu/snappy-playpen/pull/187'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by didrocks** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/ubuntu/snappy-playpen/pull/187'>Refresh</a>.<a href='#crh-update-none'></a>

File: hexchat/snapcraft.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: hexchat
-version: "2.12.1"
-summary: IRC client for X based on X-Chat 2
+version: 2.12.1
```
Comment (didrocks) [inline hexchat/snapcraft.yaml]: We have as best practice to enforce version being in a string field. (that prevents if we update to "2.20" to be translated in "2.2". Please revert that change.

File: hexchat/snapcraft.yaml [hunk 2]
```diff
@@ -11,27 +11,26 @@ confinement: strict
...
   hexchat:
     command: desktop-launch $SNAP/bin/hexchat
-    plugs: [network, network-bind, x11, unity7]
+    plugs:
+      - unity7
+      - gsettings
+      - pulseaudio
+      - network
+      - network-bind
+      - home
```
Comment (didrocks) [inline hexchat/snapcraft.yaml]: This was less than 80 characters, same, as per our best practices, please leave current list in one line as it is for clarity.

File: hexchat/snapcraft.yaml [hunk 3]
```diff
@@ -11,27 +11,26 @@ confinement: strict
...
 parts:
   hexchat:
```
Comment (didrocks) [inline hexchat/snapcraft.yaml]: Same, as part of our best practices (sorry, that should be documented), the order here should be:
```
  source:
  plugin:
  snap:
  build-packages:
  after:
```
We don't order them in ascii order, but in a logical one.

File: hexchat/snapcraft.yaml (hunk 1 above)
Comment (diddledan) [inline hexchat/snapcraft.yaml]: :+1:<a href='#crh-update-none'></a>

File: hexchat/snapcraft.yaml (hunk 2 above)
Comment (diddledan) [inline hexchat/snapcraft.yaml]: :+1:<a href='#crh-update-none'></a>

File: hexchat/snapcraft.yaml (hunk 3 above)
Comment (diddledan) [inline hexchat/snapcraft.yaml]: :+1:<a href='#crh-update-complete'></a>

**Comment by dholbach** [comment]
Nice work!

**Pull request merged**